test/pg.ddl.js

```diff
@@ -39,299 +33,16 @@ const test = config => {
-    },
-  },
-  expectedSql: `${defaultDomainsSql}
--- Category: Schema ------------------------------------------------------------
-
-CREATE TABLE "Schema" (
-  "Id"            bigserial,
-  "logicalField"  boolean,
-  "textField"     text,
-  "bigintField"   bigint,
-  "floatingField" double precision,
-  "dateField"     timestamp with time zone,
-  "bytesField"    bytea,
-  "fnField"       text,
-  "timeField"     time with time zone,
-  "dateDayField"  date,
-  "dateTimeField" timestamp with time zone,
-  "idField"       bigint,
-  "jsonField"     jsonb
-);
-
-ALTER TABLE "Schema" ADD CONSTRAINT "pkSchemaId" PRIMARY KEY ("Id");`,
-});
-
-test({
-  name: 'generateDDL with required fields',
-  schemas: {
-    Schema: {
-      required1: { domain: 'Nomen', required: true },
-      required2: { domain: 'Nomen', required: true },
-      field: { domain: 'Nomen' },
-    },
-  },
-  expectedSql: `${defaultDomainsSql}
--- Category: Schema ------------------------------------------------------------
-
-CREATE TABLE "Schema" (
-  "Id"        bigserial,
-  "required1" text NOT NULL,
-  "required2" text NOT NULL,
-  "field"     text
-);
-
-ALTER TABLE "Schema" ADD CONSTRAINT "pkSchemaId" PRIMARY KEY ("Id");`,
-});
-
-test({
-  name: 'generateDDL with default values',
-  schemas: {
-    Schema: {
-      default1: { domain: 'Nomen', default: 'default value1' },
-      default2: { domain: 'Nomen', default: 'default value2' },
-      field: { domain: 'Nomen' },
-    },
-  },
-  expectedSql: `${defaultDomainsSql}
--- Category: Schema ------------------------------------------------------------
-
-CREATE TABLE "Schema" (
-  "Id"       bigserial,
-  "default1" text DEFAULT 'default value1',
-  "default2" text DEFAULT 'default value2',
-  "field"    text
-);
-
-ALTER TABLE "Schema" ADD CONSTRAINT "pkSchemaId" PRIMARY KEY ("Id");`,
-});
-
-test({
-  name: 'generateDDL with Global',
-  schemas: {
-    GlobalEntity1: registry({
-      field: { domain: 'Nomen' },
-    }),
-    GlobalEntity2: registry({
-      field: { domain: 'Nomen' },
-      globalEntity: { category: 'GlobalEntity1' },
-    }),
-  },
-  expectedSql: `${defaultDomainsSql}
--- Category: GlobalEntity1 -----------------------------------------------------
-
-CREATE TABLE "GlobalEntity1" (
-  "Id"    bigint,
-  "field" text
-);
-
-ALTER TABLE "GlobalEntity1" ADD CONSTRAINT "pkGlobalEntity1Id" \
-PRIMARY KEY ("Id");
-
--- Category: GlobalEntity2 -----------------------------------------------------
-
-CREATE TABLE "GlobalEntity2" (
-  "Id"           bigint,
-  "field"        text,
-  "globalEntity" bigint
-);
-
-ALTER TABLE "GlobalEntity2" ADD CONSTRAINT "pkGlobalEntity2Id" \
-PRIMARY KEY ("Id");`,
-});
+[
+  'default',
+  'required',
+  'defaultDomains',
+  'enum',
+  'master',
+  'many',
+  'domainTypes',
+  'foreignKeyBeforeCategory',
+  'globalCategories',
+].map(cfgName => require(__dirname + '/pg.ddl.' + cfgName))
+  .map(cfg => cfg(metaschema.decorators))
+  .forEach(test);
```

Comment:
I'd inline `test` here, moreover, I'd prefer to have a default SQL placed somewhere, and for `test` function to prepend it to an expected SQL so that there is no need to write it every config.